devtools-ci-index.yaml

```diff
@@ -3147,3 +3147,20 @@
             git_repo: fabric8-test
             ci_project: 'devtools'
             timeout: '30m'
+        - '{ci_project}-{git_repo}-fabric8-analytics':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-api-gateway
```

Comment:
The git repo is `fabric8-analytics-api-gateway`: https://github.com/fabric8-analytics/fabric8-analytics-api-gateway